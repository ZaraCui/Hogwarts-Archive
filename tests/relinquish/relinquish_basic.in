ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Hermione Granger
RENT 100000 1003
RELINQUISH 100000 1003
EXIT

