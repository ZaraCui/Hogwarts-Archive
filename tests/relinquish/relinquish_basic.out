user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: Success.

user: Ending Archive process.
